Form/FormFlow.php

```diff
@@ -619,6 +640,17 @@ protected function bindFlow() {
 			$reset = true;
 		}
 
+		if (in_array($this->getRequest()->getMethod(), array('POST', 'PUT')) && ! $this->dataManager->exists($this)) {
+			// Form is expired, kill posted data and reset
```

Comment:
Form is expired, kill posted data and reset -> flow is expired, drop posted data and reset